scripts/build.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
 BUILD_TIME=$(date -u +%m-%d-%Y.%H-%M-%S.UTC)
-VERSION=$(cat version/CURRENT_VERSION | tr -d \\)
+VERSION=$(cat version/CURRENT_VERSION | tr -d '\n')
 PKG_NAME=github.com/contiv/netplugin/version
 
+if [ -n "$BUILD_VERSION" ]; then
```

Comment:
I find this a bit confusing to follow this flow. VERSION is defined first, then it is replaced by BUILD_VERSION, then BUILD_VERSION is defined again based on VERSION. If nothing else, it requires a bit of commenting to clarify what's defined and used.